### PR TITLE
Vulcan: Format wikitext headings

### PR DESCRIPTION
**Pull request by andyg0808**
Add styles to get wikitext headings looking like the [mockup](https://www.figma.com/file/jRG4EyQXRDMGfyLqut08KP/iFixit---Working-files?node-id=9318%3A156345&t=0OGkl5ifkNp7U5YO-0).

## QA Notes
You should be able to see what the headings look like on the
/Vulcan/troubleshooting-test#Section_h3_A_sub_section_heading
page.

Closes ifixit/ifixit#46247

### CODE REVIEW

**Comment by deltuh-vee** [comment]
dev_block ✊ 
I'm getting this error when visiting /Vulcan/troubleshooting-test
image

**Comment by deltuh-vee** [comment]
The headings look a little more styled now, but are they supposed to have separating lines?
image

**Comment by andyg0808** [comment]
un_dev_block :+1: 
The mockup doesn't show separating lines, so I'm pretty sure we don't need them:
image

**Comment by deltuh-vee** [comment]
QA 🎬 
Headings looks similar to the mockup.

**Comment by batbattur** [comment]
dev_block 🚛 looks like there's a merge conflict

**Comment by andyg0808** [comment]
un_dev_block :+1:

**Comment by deltuh-vee** [comment]
Carry over QA 🎬

File: frontend/templates/troubleshooting/index.tsx [hunk 1]
```diff
@@ -62,11 +68,53 @@ function ConclusionSection({ conclusion }: { conclusion: Problem }) {
...
+}: {
+   html: string;
+   className?: string;
+}) {
+   const renderStyles: SystemStyleObject = {
+      '.headerContainer': {
+         display: 'flex',
+         alignItems: 'baseline',
+         marginBottom: 2,
+      },
```
Comment (jarstelfox) [inline frontend/templates/troubleshooting/index.tsx:88]: Do you know if chakra automagically does scoping?

i.e. `& .headerContainer`?

**Review by jarstelfox — APPROVED** [review]
CR 🌵

File: frontend/templates/troubleshooting/index.tsx (hunk 1 above)
Comment (andyg0808) [reply frontend/templates/troubleshooting/index.tsx:88]: I believe it does:
```css
.css-eec6dr .headerContainer {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: baseline;
  -webkit-box-align: baseline;
  -ms-flex-align: baseline;
  align-items: baseline;
  margin-bottom: var(--chakra-space-2);
}
```
(from the rendered page)

**Pull request merged**